### DISCUSSION TITLE
zsh package is not shipped with zsh-newuser-install

### DISCUSSION THREAD

**Discussion by gphg** [termux/termux-packages#20603] [Q&A] [answered]
Command `zsh-newuser-install` is not found. I have to do a tricky method to get zsh configurated and working properly:
* Has working history 
* Better prompt
* Produces `~/.zshrc`

Is this by design?

**Comment by TomJo2000** [comment, marked as answer]
Closed in:
- #20604

---

Original answer:

`zsh-newuser-install` is part of the `zsh/newuser` module.
We do not currently compile this in as it hasn't previously been requested.
https://github.com/termux/termux-packages/blob/47e87d6905408a1c657541fa9fa2355bec6deda5/packages/zsh/build.sh#L74-L90

I can open a PR to add the appropriate module to the list shortly.
